.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I wouldn't commit that file. Most of us are using phpstorm, but we should decide how/if we add those.